src/expr.rs

```diff
@@ -129,7 +129,7 @@ mod parsing {
         // https://docs.rs/winnow/latest/winnow/_topic/language/index.html
         (
             one_of(|c: char| c.is_alpha() || c == '_'),
-            take_while(0.., |c: char| c.is_alphanum() || c == '_'),
+            take_while(0.., |c: char| c.is_alphanum() || c == '_' || c == '.'),
```

Comment:
Sorry, but I also included this by mistake that allows variable names with dots that I meant to send as a separate commit/PR.